bin/city

```diff
@@ -5,15 +5,20 @@ from importlib import import_module
 
 _, *args = argv
 
+city_dirs = [ 'cities', 'calib' ]
+
 city_name = args[0]
 
-try:
-    module_name = 'invisible_cities.cities.' + city_name
-    city_class  = getattr(import_module(module_name), city_name.capitalize())
-except ModuleNotFoundError:
-    print('available cities: <TODO look at contents of config directory>')
-    print('usage TODO: <get this from our config parser>')
-    traceback.print_exc()
-    exit(1)
-else:
-    city_class.drive(args)
+for i, fldr in enumerate(city_dirs):
```

Comment:
We use the name 'directory' to refer to directories in IC, so please avoid introducing a different term for it: 'folder'. Please spell it `for dir_ in city_dirs`. The trailing underscore ensures that we avoid shadowing a built-in. Alternatively, `directory` would be fine too.

But, more importantly, is there any good reason why this city can't live with all the other ones? Do we really need to introduce an extra place where cities might live?

This touches on something else that has been bothering me with this city: it seems to introduce modifications to parts of IC where I wouldn't normally expect authors of new cities to have to go. (This is just the impression I get after a cursory look: I haven't had the time to look at it in any detail.)

*If* (and it's a big *if*) the need to add this extra directory turns out to be genuine, then we can do without the `enumerate` and the `if i == len(city_dirs)-1`. The correct spelling of that idea is to cut the old body of the `NoModuleFoundError` handler, replacing it with `pass` or `continue`, and to move it to the `else` block of the `for loop` (yes, the `for loop`!) The addition of `break` to the `else` block of the `try` will still be necessary.